.github/workflows/check-plugin.yml

```diff
@@ -88,11 +88,25 @@ jobs:
             pip freeze
       - name: Install pre-commit hooks
         run: |
-            cd ${{ inputs.plugin }}
             pre-commit install --install-hooks
             pre-commit install --hook-type pre-push
       - name: Run linter
-        run: make plugin=${{ inputs.plugin }} lint
+        run: |
+          pre-commit run trailing-whitespace --all-files
+          pre-commit run end-of-file-fixer --all-files
+          pre-commit run check-yaml --all-files
+          pre-commit run check-added-large-files --all-files
+          pre-commit run check-case-conflict --all-files
+          pre-commit run check-merge-conflict --all-files
+          pre-commit run debug-statements --all-files
+          pre-commit run requirements-txt-fixer --all-files
+          pre-commit run flake8 --all-files
+          pre-commit run isort --all-files --hook-stage manual
+          pre-commit run black --all-files --hook-stage manual
+          pre-commit run secret_scan --all-files --hook-stage manual
+          pre-commit run bandit --all-files --hook-stage manual
+          pre-commit run pylint-${{inputs.plugin}} --all-files --hook-stage manual
+
```

Comment:
Is the `make lint` still useful?

Reply:
I will update the `make lint` to run these instead

Reply:
Updated `make lint` it will now run each individual pre-commit hook with `&&` the drawback from this is that it will stop running if one hook fails. The alternative is to use `;` but CI won't detect if a hook fails so I've gone with `&&`.

```
lint:
	pre-commit run trailing-whitespace --all-files && pre-commit run end-of-file-fixer --all-files && pre-commit run check-yaml --all-files && pre-commit run check-added-large-files --all-files && pre-commit run check-case-conflict --all-files && pre-commit run check-merge-conflict --all-files && pre-commit run debug-statements --all-files && pre-commit run requirements-txt-fixer --all-files && pre-commit run flake8 --all-files && pre-commit run isort-$(plugin) --all-files --hook-stage manual && pre-commit run black-$(plugin) --all-files --hook-stage manual && pre-commit run secret_scan --all-files --hook-stage manual && pre-commit run bandit --all-files --hook-stage manual && pre-commit run pylint-$(plugin) --all-files --hook-stage manual && pre-commit run pylint-$(plugin)-features --all-files --hook-stage manual && pre-commit run pylint-$(plugin)-tests --all-files --hook-stage manual
```